src/invidious/user/imports.cr

```diff
@@ -53,13 +47,8 @@ struct Invidious::User
         privacy = PlaylistPrivacy::Private
       end
 
-      if title && privacy && user
-        playlist = create_playlist(title, privacy, user)
-      end
-
-      if playlist && description
-        Invidious::Database::Playlists.update_description(playlist.id, description)
-      end
+      playlist = create_playlist(title, privacy, user)
+      Invidious::Database::Playlists.update_description(playlist.id, description)
 
       # Add each video to the playlist from the body content
       CSV.each_row(raw_body) do |row|
```

Comment:
I'd recommend to read the body using the following way:
```crystal
csv_body = CSV.new(raw_body, headers: true)
csv_body.each do |row|
  # Put the rest of the code here
end
```

By doing so, you remove a lot of useless checks (the first line is always headers, so it's skipped in the `.new` statement, and if there is no other CSV lines (no videos), `.each` will be skipped)

Reply:
Updates made, checked, and commited.